### ISSUE TITLE
Homegear-MAX: Support Auto-ACK for window contacts

### ISSUE THREAD

**Issue by andyboeh** [Homegear/Homegear#353]
The MAX door/window contacts are very sensitive regarding the timing of the ACK message. Even with a responseDelay of `0`, I can't get my window contacts to reliably transmit (i.e. most of the time, they blink 3 times, indicating a transmission error). The Homegear log file clearly shows multiple received packets.

To overcome this limit, culfw and a-culfw (I'm using a-culfw on a MAX!Cube) have an auto-ack feature implemented for certain device types (https://github.com/heliflieger/a-culfw/blob/f4305ea7ca9aba2ace6978c9c29e2645072e5c66/culfw/clib/rf_moritz.c#L152). The CUL/CUN needs to know the MAX address to use, then Homegear does not need to send the ack (I had a look at fhem for reference: https://github.com/fhem/fhem-mirror/blob/3856444938cab1e7ea46349d329a7c717ca33e8c/fhem/FHEM/14_CUL_MAX.pm#L165).

Could you add support for Auto-ACK?

**Comment by hfedcba** [comment]
Yes, good idea.

**Comment by Flole998** [comment]
Is this also an issue for cc1101 directly attached to a raspberry pi for example? It should be much faster than the USB CUL.

**Comment by andyboeh** [comment]
No, since a directly attached CC1101 has no firmware that could send the Auto-ACK.

**Comment by Flole998** [comment]
Of course not, but does it respond fast enough so it's not needed there?
